apps/site/lib/site/content_rewriters/liquid_objects/fare.ex

```diff
@@ -236,4 +279,13 @@ defmodule Site.ContentRewriters.LiquidObjects.Fare do
       Keyword.put(args, k, String.to_existing_atom(v))
     end)
   end
+
+  # Fill in any missing/required arguments with the default,
```

Comment:
why did this move?

Reply:
I typically like to have these secondary functions in the order they are called in the pipeline. filter_insert occurs before get_fares.